recipe/meta.yaml

```diff
@@ -30,7 +30,7 @@ requirements:
     - dask-core ==2023.1.1
     - distributed ==2023.1.1
     - dask ==2023.1.1
-    - pynvml >=11.0.0
+    - pynvml >=11.0.0,<11.5
```

Comment:
This is the only real change

https://github.com/rapidsai/dask-cuda/compare/v23.02.00...v23.02.01

Trying to fix an issue encountered with the new release of `pynvml`

We may want to hot-fix the old `dask-cuda` packages in conda-forge to contain this constraint as well

Reply:
For more context a future `dask-cuda` version will be able to relax this constraint due to this fix ( https://github.com/rapidsai/dask-cuda/pull/1118 ), but older `dask-cuda` versions lack the fix so need the constraint ( https://github.com/rapidsai/dask-cuda/pull/1123 )

Reply:
Submitted hot-fix for the older packages with PR ( https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/405 )